internal/config/config_test.go

```diff
@@ -473,7 +500,6 @@ func TestWebConfig(t *testing.T) {
 	t.Run("valid web config", func(t *testing.T) {
 		tempWebConfig, err := os.CreateTemp("", "temp_*web.yml")
 		assert.NoError(t, err, "cannot create temp file")
-		defer os.Remove(tempWebConfig.Name())
```

Comment:
Lets add it to cleanup instead `t.Cleanup()` so that if test fails in between the file is cleanup.